db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```

Comment:
In general it is best to remove autogenerated comments unless they are being used as some sort of documentation or a note to explain why you did a weird thing to future generations

Reply:
Makes sense.  Mt first instinct is to remove the just example portion --  the introductory portion seems relevant as basic file documentation.